public/scss/_app-error.scss

```diff
@@ -1,12 +1,13 @@
 .app-error {
-    background: rgba(255,255,255,0.8);
+    background: rgba(255,255,255,0.9);
     bottom: 0;
     height: 100%;
     left: 0;
+    padding: 10px;
     position: absolute;
     right: 0;
     top: 0;
-    z-index: 10;
+    z-index: 30;
```

Comment:
something for the future (maybe): http://www.sitepoint.com/better-solution-managing-z-index-sass/ This seems like a better way to do handle z-index.